.github/workflows/notification.yml

```diff
@@ -0,0 +1,66 @@
+name: Check CMFGEN data update workflow
+
+on:
+  schedule:
+    - cron: '0 0 * * 5' 
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK }}
+
+    steps:
+      - uses: act10ns/slack@v2.0.0
+        with:
+          status: starting
+          message: Starting Check CMFGEN data update workflow
+        if: always()
+        
+      - name: Checkout github repo
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+          fetch-depth: 0
+
+      - name: Configure git
+        run: |
+          git config --global user.name "GitHub Actions"
+          git config --global user.email "actions@github.com"
+                
+      - name: Make temporary directories
+        id: setup
+        run: |
+          mkdir ~/temp
+          
+      - name: Delete the previous data
+        id: delete_previous_data
+        run: |
+          rm -rf ./atomic 
+
+      - name: Download data
+        id: download_data
+        run: |
+          SOURCE_URL=https://kookaburra.phyast.pitt.edu/hillier/cmfgen_files/atomic_data_15nov16.tar.gz
+          wget -q -U "Mozilla/4.0" $SOURCE_URL -O ~/temp/atomic.tar.gz
+          tar -zxf ~/temp/atomic.tar.gz 
+          
+      - name: Commit files to git
+        id: commit_changes
+        run: |
+          git add .
+          git commit -m "New dataset added"
```

Comment:
I guess this fails when no changes are present. But that doesn’t mean something is wrong, right?

Reply:
Perhaps add a chained job which indicates when the new data is changed but is skipped whenever the data comparison passes? 
So if the data comparison fails this would look  like

- comparison- failed
- commit- check


In most of the circumstances- 
- comparison- passed
- commit- skipped


The commit need not be in the second job, but just a job which checks outputs of the commit step, you've already done something like this I guess

Reply:
Okay, I will do this, but first, let's see what @wkerzendorf has to say about this.

Reply:
@atharva-2001 does it really fail?